README.md

```diff
@@ -191,8 +191,63 @@ Packages found only in gcr.io/google-appengine/python:2017-06-29-190410: None
 Version differences: None
 
 ```
+## Example Run with json post-processing
```

Comment:
I think it's probably fine to include the python code you used to process. anything that makes this easier for people to start using is a good thing imo :) but I also agree with @cftorres, it might also be useful to show a bit of the json that the script consumes to produce that final output. don't think I'd put the whole thing though, we don't want to pollute the readme